packages/orbit-components/src/CountryFlag/README.md

```diff
@@ -16,18 +16,18 @@ After adding import into your project you can use it simply like:
 
 Table below contains all types of the props available in CountryFlag component.
 
-| Name     | Type                                             | Default       | Description                          |
-| :------- | :----------------------------------------------- | :------------ | :----------------------------------- |
-| code     | [`enum`](#enum)                                  | `"undefined"` | Code for the displayed country flag. |
-| dataTest | `string`                                         |               | Optional prop for testing purposes.  |
-| id       | `string`                                         |               | Set `id` for `CountryFlag`           |
-| name     | `string`                                         |               | The name for the flag.               |
-| role     | `React.HTMLAttributes<HTMLImageElement>["role"]` | `"img"`       | ARIA role for the flag image.        |
-| size     | [`enum`](#enum)                                  | `"medium"`    | The size of the CountryFlag.         |
+| Name     | Type                                             | Default       | Description                                |
+| :------- | :----------------------------------------------- | :------------ | :----------------------------------------- |
+| code     | [`enum`](#enum)                                  | `"undefined"` | Code for the displayed country flag.       |
+| dataTest | `string`                                         |               | Optional prop for testing purposes.        |
+| id       | `string`                                         |               | Set `id` for `CountryFlag`                 |
+| name     | `string`                                         | `""`          | Defines the `alt` attribute for the image. |
+| role     | `React.HTMLAttributes<HTMLImageElement>["role"]` |               | ARIA role for the flag image.              |
```

Comment:
Is there any reason why we don't set the `presentation` role as a default value? these flags are usually used as a "decorator", aren't they? 🤔

Reply:
Having `name` with default value `""` is enough. Applying `role="presentation"` is redundant

Reply:
I know, but we did the same approach in the illustrations components. So I thought we want to keep same modus operandi everywhere. :D

Reply:
According to [this](https://kb.daisy.org/publishing/docs/html/images-deco.html#:~:text=Authors%20may%20use%20the%20role%20presentation%20in%20place%20of%20an%20empty%20alt%20attribute%2C%20but%20this%20method%20is%20not%20well%20supported.), role is not as supported as empty alt. It's ok to have any approach, really.